src/Nowruz/modules/credentials/KYB/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { DescriptionModal } from './descriptionModal';
+import { UploadModal } from './uploadModal';
+import { SuccessModal } from './successModal';
+
+interface KYBModalProps {
+  open: boolean;
+  setOpen: (val: boolean) => void;
+}
+export const KYBModal: React.FC<KYBModalProps> = ({ open, setOpen }) => {
+  const [openDescModal, setOpenDescModal] = useState(true);
```

Comment:
isn't it better to reduce your states to one state with {name, open} property for better readability?
`{name: 'desc' | 'upload' | 'success', open: boolean}`